### PR TITLE
Feature/division line

### PR DESCRIPTION
**Pull request by piloshachar14**

### CODE REVIEW

File: apps/client/src/components/criminals/criminals.module.css [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
-  gap: 1em;
-  padding-left: 1em;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5em;
   padding-right: 1em;
-  padding-bottom: 1em;
+  padding-left: 1em;
+}
+.criminalsDiv {
```
Comment (SeanCarmel) [inline apps/client/src/components/criminals/criminals.module.css]: I think you should change className.

File: apps/client/src/components/criminals/criminals.module.css [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+  display: grid;
+  grid-template-columns: auto auto;
+  grid-template-rows: auto auto;
+  gap: 0.5em;
+  padding: 0.5em;
+  height: 90%;
+}
+.divider {
+  width: 100%;
+  height: 1px;
```
Comment (SeanCarmel) [inline apps/client/src/components/criminals/criminals.module.css]: I think you should change px to rem/em/%.

File: apps/client/src/components/criminals/criminals.tsx [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { useState } from 'react';
 import styles from './criminals.module.css';
-import { Card } from '../';
+import { Card, Category } from '../';
+import BlockTwoToneIcon from '@mui/icons-material/BlockTwoTone';
```
Comment (SeanCarmel) [inline apps/client/src/components/criminals/criminals.tsx]: I think `import {BlockTwoTone} from '@mui/icons-material';` is better.

File: apps/client/src/components/criminals/criminals.tsx [hunk 4]
```diff
@@ -22,27 +23,64 @@ export const Criminals: React.FC = () => {
...
+        style={{ height: '32vh', border: 'none' }}
+      >
+        <div className={styles.criminalsDiv}>
+          {criminals.map((criminal, index) => (
+            <Card
+              key={index}
+              title={criminal.name}
+              description=":שתיות מפשיעות
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+              stringBorder="2px rgb(200, 22, 22) solid"
```
Comment (SeanCarmel) [inline apps/client/src/components/criminals/criminals.tsx]: I think you should not use px.

File: apps/client/src/components/criminals/criminals.tsx [hunk 5]
```diff
@@ -22,27 +23,64 @@ export const Criminals: React.FC = () => {
...
-          stringBorder="2px rgb(200, 22, 22) solid"
-          descriptionStyle={{
-            fontSize: 'small',
-            textAlign: 'end',
-            paddingRight: '1em',
-            paddingBottom: '0.5em',
-          }}
-        />
-      ))}
+              stringBorder="2px white solid"
```
Comment (SeanCarmel) [inline apps/client/src/components/criminals/criminals.tsx]: I think px is not reccomended.

File: apps/client/src/components/toasts/toasts.module.css [hunk 6]
```diff
@@ -1,24 +1,29 @@
...
   display: grid;
   grid-template-columns: repeat(2, 1fr);
-  grid-template-rows: repeat(3, 1fr);
+  grid-template-rows: repeat(2, 1fr);
   gap: 1em;
   padding: 1em;
 }
+.divider {
+  width: 100%;
+  height: 1px;
```
Comment (SeanCarmel) [inline apps/client/src/components/toasts/toasts.module.css]: maybe don't use px?

File: apps/client/src/components/criminals/criminals.module.css [hunk 7]
```diff
@@ -1,9 +1,22 @@
...
+  grid-template-columns: auto auto;
+  grid-template-rows: auto auto;
+  gap: 0.5em;
+  padding: 0.5em;
+  height: 90%;
+}
+.divider {
+  width: 100%;
+  height: 0.1em;
+  background-color: rgb(224, 219, 219);
```
Comment (piloshachar14) [inline apps/client/src/components/criminals/criminals.module.css:20]: add this color in root

File: apps/client/src/components/criminals/criminals.tsx [hunk 8]
```diff
@@ -22,27 +23,64 @@ export const Criminals: React.FC = () => {
     { name: 'תומר שטיינברג' },
     { name: 'תומר שטיינברג' },
     { name: 'תומר שטיינברג' },
+    { name: 'תומר שטיינברג' },
+    { name: 'תומר שטיינברג' },
+  ]);
+  const [personaNonGratas, setpersonaNonGratas] = useState<User[]>([
+    { name: 'hifehn' },
+    { name: 'hifehn' },
```
Comment (piloshachar14) [inline apps/client/src/components/criminals/criminals.tsx:31]: dont keep hard coded values

File: apps/client/src/components/criminals/criminals.tsx [hunk 9]
```diff
@@ -22,27 +23,64 @@ export const Criminals: React.FC = () => {
...
     <div className={styles.criminals}>
-      {criminals.map((criminal, index) => (
-        <Card
-          key={index}
-          title={criminal.name}
-          desc=":שתיות מפשיעות
+      <div className={styles.divider} />
+      <Category
+        title=":בני עוולה רגילים"
+        style={{ height: '32vh', border: 'none' }}
```
Comment (piloshachar14) [inline apps/client/src/components/criminals/criminals.tsx:41]: don't use vh, use %

Comment (piloshachar14) [reply apps/client/src/components/criminals/criminals.tsx:41]: and border='none' is not the default value ?

File: apps/client/src/components/criminals/criminals.tsx [hunk 10]
```diff
@@ -22,27 +23,64 @@ export const Criminals: React.FC = () => {
...
+      >
+        <div className={styles.criminalsDiv}>
+          {criminals.map((criminal, index) => (
+            <Card
+              key={index}
+              title={criminal.name}
+              description=":שתיות מפשיעות
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+              stringBorder="0.1em rgb(200, 22, 22) solid"
```
Comment (piloshachar14) [inline apps/client/src/components/criminals/criminals.tsx:50]: add color to root

File: apps/client/src/components/criminals/criminals.tsx [hunk 11]
```diff
@@ -22,27 +23,64 @@ export const Criminals: React.FC = () => {
...
+                paddingBottom: '0.5em',
+              }}
+            />
+          ))}
+        </div>
+      </Category>
+      <div className={styles.divider} />
+      <Category
+        title=":פרסונה נון גרטה לא עלינו"
+        style={{ height: '32vh', border: 'none' }}
```
Comment (piloshachar14) [inline apps/client/src/components/criminals/criminals.tsx:64]: dont use vh but %

Comment (piloshachar14) [reply apps/client/src/components/criminals/criminals.tsx:64]: check if border is necessary

File: apps/client/src/components/toasts/toasts.module.css [hunk 12]
```diff
@@ -1,24 +1,29 @@
 .toasts {
   display: flex;
   flex-direction: column;
-  gap: 1em;
-  padding: 1em;
+  gap: 0.5em;
+  padding-right: 1em;
+  padding-left: 1em;
```
Comment (piloshachar14) [inline apps/client/src/components/toasts/toasts.module.css:6]: you can put padding: 0 1em

File: apps/client/src/components/toasts/toasts.module.css [hunk 13]
```diff
@@ -1,24 +1,29 @@
...
   grid-template-columns: repeat(2, 1fr);
-  grid-template-rows: repeat(3, 1fr);
+  grid-template-rows: repeat(2, 1fr);
   gap: 1em;
   padding: 1em;
 }
+.divider {
+  width: 100%;
+  height: 0.1em;
+  background-color: rgb(224, 219, 219);
```
Comment (piloshachar14) [inline apps/client/src/components/toasts/toasts.module.css:19]: add color to root

File: apps/client/src/components/toasts/toasts.tsx [hunk 14]
```diff
@@ -46,28 +58,38 @@ export const Toasts: React.FC<Props> = ({ isLoggedIn }) => {
     <div className={styles.toasts}>
       {isLoggedIn ? (
         <>
-          <Category title=":שתיות קרובות" style={{ height: '34vh' }}>
+          <div className={styles.divider} />
+          <Category
+            title=":שתיות קרובות"
+            style={{ height: '32vh', border: 'none' }}
```
Comment (piloshachar14) [inline apps/client/src/components/toasts/toasts.tsx:64]: % instead of vh, border necessary ?

File: apps/client/src/components/toasts/toasts.tsx [hunk 15]
```diff
@@ -46,28 +58,38 @@ export const Toasts: React.FC<Props> = ({ isLoggedIn }) => {
     <div className={styles.toasts}>
       {isLoggedIn ? (
         <>
-          <Category title=":שתיות קרובות" style={{ height: '34vh' }}>
```
Comment (piloshachar14) [inline apps/client/src/components/toasts/toasts.tsx:49]: % instead of vh

File: apps/client/src/components/toasts/toasts.tsx [hunk 16]
```diff
@@ -46,28 +58,38 @@ export const Toasts: React.FC<Props> = ({ isLoggedIn }) => {
...
               ))}
             </div>
           </Category>
-          <Category title=":שתיות שעברו" style={{ height: '34vh' }}>
+
+          <div className={styles.divider} />
+
+          <Category
+            title=":שתיות שעברו"
+            style={{ height: '32vh', border: 'none' }}
```
Comment (piloshachar14) [inline apps/client/src/components/toasts/toasts.tsx:83]: % instead of vh

File: apps/client/src/components/toasts/toasts.tsx [hunk 17]
```diff
@@ -46,28 +58,38 @@ export const Toasts: React.FC<Props> = ({ isLoggedIn }) => {
...
                   toastsDate={toast.toastDate}
-                  desc={toast.toastDesc}
-                  stringBorder="2px chartreuse solid"
+                  description={toast.toastDesc}
+                  stringBorder="0.1em chartreuse solid"
                 />
               ))}
             </div>
           </Category>
-          <Category title=":שתיות שעברו" style={{ height: '34vh' }}>
```
Comment (piloshachar14) [inline apps/client/src/components/toasts/toasts.tsx:62]: % instead of vh

File: apps/client/src/components/toasts/toasts.tsx [hunk 18]
```diff
@@ -46,28 +58,38 @@ export const Toasts: React.FC<Props> = ({ isLoggedIn }) => {
...
+            style={{ height: '32vh', border: 'none' }}
+          >
             <div className={styles.toastsContainerLoggedIn}>
               {happendToasts.map((toast, index) => (
                 <Card
                   key={index}
                   title={toast.user.name}
                   toastsDate={toast.toastDate}
-                  desc={toast.toastDesc}
-                  stringBorder="2px rgb(200, 22, 22) solid"
```
Comment (piloshachar14) [inline apps/client/src/components/toasts/toasts.tsx:70]: add color to root

**Pull request merged**